readme.md

```diff
@@ -4,33 +4,36 @@
 [![Discord Follow](https://dcbadge.vercel.app/api/server/MsKAeKF8kU?style=flat)](https://discord.gg/MsKAeKF8kU)
 [![Twitter Follow](https://img.shields.io/twitter/follow/fixieai?style=social)](https://twitter.com/fixieai)
 
-## What is AI.JSX?
+## About AI.JSX
 
-AI.JSX is a framework for building AI applications using Javascript and [JSX](https://react.dev/learn/writing-markup-with-jsx). While AI.JSX [is not React](https://docs.ai-jsx.com/is-it-react), it's designed to look and feel very similar while also integrating seamlessly with React-based projects. With AI.JSX, you don't just use JSX to describe what your UI should look like, you also use it to describe how **Large Language Models**, such as ChatGPT, should integrate into the rest of your application. The end result is a powerful combination where _intelligence_ can be deeply embedded into the application stack.
+AI.JSX is a framework for building AI applications using Javascript and JSX. With AI.JSX, you can easily orchestrate multiple Large Language Model (LLM) calls expressed as modular, reusable components, and interweave UI components with your AI components. This means you can rely on the LLM to construct your UI dynamically from a set of provided React components. AI.JSX also provides native support for tools, Document Question & Answering, and much more.
```

Comment:
I actually think we we can simplify this paragraph even more.

Instead of "orchestrate multiple LLM calls expressed as..." let's just call it what it is: great tools for prompt engineering. Instead of "interweave UI components with AI components", let's just say you can have the LLM render React components in its response instead of just text.

Basically, I think we should evolve this to the eli5 version

Reply:
💯 for the "above the fold README" being ELI5.

Reply:
just committed an update to simplify per Zach's feedback...thoughts?